src/include/sof/coherent.h

```diff
@@ -69,6 +69,14 @@ struct coherent {
 	struct list_item list;	/* coherent list iteration */
 } __coherent;
 
+#if CONFIG_INCOHERENT
```

Comment:
Do we need to document this configuration somewhere?

Reply:
The documentation for every `CONFIG_` can be found at:

```
git grep -A10 INCOHERENT '*/Kconfig'

src/platform/Kconfig:config INCOHERENT
src/platform/Kconfig-   bool "Enable cached data access via the Coherent API"
src/platform/Kconfig-   default y if CAVS || ACE
src/platform/Kconfig-   default n
src/platform/Kconfig-   help
src/platform/Kconfig-     The architecture is cache incoherent. i.e FW has to manually manage
src/platform/Kconfig-     cache coherency amongst objects that are used on multiple cores.
src/platform/Kconfig-     This setting should only be disabled for cache incoherent
src/platform/Kconfig-     architectures for testing without cached access to shared data.


```

Reply:
Also at:

```
ninja -C build-mtl menuconfig
/
INCOHERENT
```

```